packages/lab/stories/date-input/date-input.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import { DateInput, DateInputProps } from "@salt-ds/lab";
+import { Meta, StoryFn } from "@storybook/react";
+
+export default {
+  title: "Lab/Date Input",
+  component: DateInput,
+} as Meta<typeof DateInput>;
+
+const DateInputTemplate: StoryFn<DateInputProps> = (args) => {
+  return <DateInput {...args} />;
+};
+
+const formatter = (input: string): string => {
+  const date = new Date(input);
+  // @ts-ignore evaluating validity of date
+  return isNaN(date)
+    ? input
+    : new Intl.DateTimeFormat("en-US", {
+        year: "numeric",
+      }).format(date);
+};
+
+export const Default = DateInputTemplate.bind({});
+Default.args = {};
+
+export const CustomFormatter = DateInputTemplate.bind({});
```

Comment:
I was looking at the stories and this formatting means the code view doesn't show much. Maybe including a custom formatter example in the site docs would be useful